### PR TITLE
Fixed behaviour of dropdowns

### PR DESCRIPTION
**Pull request by matzimowski**
closes https://github.com/brainly/style-guide/issues/281

screen shot 2015-09-14 at 14 08 25

its size is the same like small select:
screen shot 2015-09-14 at 14 23 15

### CODE REVIEW

File: src/components/dropdowns/_dropdowns.scss [hunk 1]
```diff
@@ -54,9 +55,18 @@ $include-html: false !default;
     }
 
     &__items {
+      position: absolute;
+      top: $dropdownHeight - 2* $dropdownBorderSize;
       visibility: hidden;
       padding: 0;
       margin: 0 0 $layoutDefaultPadding/2;
+      background-color: $white;
```
Comment (aju) [inline src/components/dropdowns/_dropdowns.scss]: Please create component specific variables for it

Comment (matzimowski) [inline src/components/dropdowns/_dropdowns.scss]: done

File: src/components/dropdowns/_dropdowns.scss [hunk 2]
```diff
@@ -64,26 +79,29 @@ $include-html: false !default;
...
     }
 
     &--opened {
-      background-color: $white;
-      border-color: $grayPrimary;
-      height: auto;
+      background-color: $dropdownBackground;
+      border-color: $dropdownOpenedBorderColor;
+      overflow: visible;
+      border-radius: 16px 16px 0 0;
```
Comment (kdzwinel) [inline src/components/dropdowns/_dropdowns.scss]: why here border-radius is not calculated?

Comment (matzimowski) [inline src/components/dropdowns/_dropdowns.scss]: done :older_man:  + :older_woman:  => :octopus:

File: src/components/dropdowns/_dropdowns.scss [hunk 3]
```diff
@@ -64,26 +79,29 @@ $include-html: false !default;
...
+      border-bottom-color: transparent;
 
       &:hover,
       &:active {
-        background-color: $white;
-        border-color: $grayPrimary;
+        background-color: $dropdownBackground;
+        border-color: $dropdownOpenedBorderColor;
+        border-bottom-color: transparent;
```
Comment (kdzwinel) [inline src/components/dropdowns/_dropdowns.scss]: border-bottom-color: transparent; is default for --opened

**Comment by aju** [comment]
Best

**Comment by kdzwinel** [comment]
test on firefox and safari and :+1:

File: src/components/dropdowns/_dropdowns.scss [hunk 4]
```diff
@@ -9,20 +13,21 @@ $include-html: false !default;
 
   .mint-dropdown {
     @include component;
+    @include hole;
```
Comment (mr-mig) [inline src/components/dropdowns/_dropdowns.scss]: I have missed this mixin somewhere...

File: src/components/dropdowns/_dropdowns.scss (hunk 3 above)
Comment (matzimowski) [inline src/components/dropdowns/_dropdowns.scss]: it overides styles for :hover,:active when there is no '--opened';
i has to be like that my dear

**Comment by matzimowski** [comment]
updated, please :+1:

**Comment by aju** [comment]
yes

**Comment by kdzwinel** [comment]
ok

**Comment by vergilius** [comment]
:+1:

**Pull request merged**